src/main/java/mate/jdbc/util/ConnectionUtil.java

```
@@ -0,0 +1,28 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load JDBC Driver for MySQL", e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
+            dbProperties.put("password", "12345678");
+            return DriverManager.getConnection("jdbc:mysql://localhost:3306/library_db",
+                    dbProperties);
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get connection to DB",e);
```

you need to use your own exception DataProcessingException

---

I don't agree, this exception connecting with data processing, not with connection.
The Readme file says "You should rethrow this exception in catch block on dao layer."